tests/test_export_mermaid.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Tests for exporting schemapacks in mermaid format."""
+
+import pytest
+
+from schemapack import load_schemapack
+from schemapack.export.mermaid import export_mermaid
+from tests.fixtures.examples import EXAMPLES_DIR, VALID_SCHEMAPACK_PATHS
+
+
+@pytest.mark.parametrize(
+    "with_properties, file_suffix",
+    [(True, "_w_props.mm.txt"), (False, "_wo_props.mm.txt")],
+)
```

Comment:
Rather take the same approach as for the schemapack and datapack examples:

- Automatically ready all erd diagram examples in the tests.fixture.examples to generate an ERD_PATHS dict
- use a naming convention so that, from the name of the erd example, the corresponding schemapack as well as the parameters (w/o props) can be deduced

Like that you can simply add more files to the example directory and they will be automatically picked up by the tests.

Reply:
Fixed in 5cfe4bcd123a257120794f27337ffa765e894b3f